package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "feathers-socket-commons",
   "description": "Shared functionality for websocket providers",
-  "version": "2.3.1",
+  "version": "2.4.0",
```

Comment:
Oh also, no version bump necessary that will happen automatically during release.